src/main/java/com/faforever/client/preferences/Preferences.java

```diff
@@ -200,4 +204,12 @@ public NewsPrefs getNews() {
   public DeveloperPrefs getDeveloper() {
     return developer;
   }
+
+  public void setGameListSorting(ObservableList<Pair<String, SortType>> gameListSorting) {
+    this.gameListSorting.set(gameListSorting);
+  }
```

Comment:
Avoid setters for list properties; if there are listeners on the list (not the property) they won't get notified about the new values and even create a memory leak (because they keep a reference on a list that is never updated)

Instead, use `getGameListSorting().setAll()` in the caller.